src/main/java/de/samply/icd10dictionary/dao/IcdCodeDaoPostgres.java

```diff
@@ -84,7 +84,8 @@ public int count() {
     final String sql =
         "SELECT code, kind, display, definition, parentCode, childCodes "
             + "FROM IcdCode "
-            + "WHERE childCodes = '' "
+            + "WHERE code like ?"
```

Comment:
It has to be `childCodes = '' AND (code like ? OR ...)` because we like to find only codes without childs.